checkov/github/dal.py

```diff
@@ -85,28 +82,34 @@ def get_organization_webhooks(self) -> dict[str, Any] | None:
         return data
 
     def get_repository_collaborators(self) -> dict[str, Any] | None:
-        data = self._request(endpoint="repos/{}/{}/collaborators".format(self.org, self.current_repository),
-                             allowed_status_codes=[200])
+        if self.org:
```

Comment:
This is not needed, org can exist (to retrieve org security info) but the repository collaborators should be queried with repo-related details, i.e. repo_owner.

Reply:
same explanation here for:
https://api.github.com/repos/KiteFlyingInTheSky/terragoat/collaborators
https://api.github.com/repos/marynaKK/terragoat/collaborators

Reply:
fixed this too :)